src/components/ConnectionStatus.module.css

```
@@ -0,0 +1,407 @@
+  transform: translateY(-50%);
+}
+.w {
+  left: -1px;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+/* --- Text --- */
+
+.label {
+  font-family: "Marcellus", Georgia, serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Marcellus" (font-family-name-quotes)

(font-family-name-quotes)